docker-jans-monolith/README.md

```diff
@@ -32,35 +42,37 @@ The following environment variables are supported by the container:
 | `MYSQL_HOST`            | MySQL host.                                      | `mysql` which is the docker compose service name |
 
 
-## Pre-requisites
+## How to run
 
-- [Docker](https://docs.docker.com/install). Docker compose should be installed by default with Docker.
+Download the compose file 
 
-## How to run
+```bash
+
+wget https://raw.githubusercontent.com/JanssenProject/jans/main/docker-jans-monolith/jans-mysql-compose.yml 
+```
+
+This docker compose file run two containers, the janssen monolith container and mysql container.
```

Comment:
`This docker compose file runs two containers, the janssen monolith container and mysql container.`